pkg/apis/cluster/mutation/mutation_test.go

```diff
@@ -123,3 +123,30 @@ func TestStandardizeClusterResourceModels(t *testing.T) {
 		}
 	}
 }
+
+func TestSetDefaultClusterResourceModels(t *testing.T) {
+	type args struct {
+		cluster *clusterapis.Cluster
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{
+			name: "test set default Cluster",
+			args: args{
+				cluster: &clusterapis.Cluster{},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			SetDefaultClusterResourceModels(tt.args.cluster)
+		})
+		modelCounts := len(tt.args.cluster.Spec.ResourceModels)
+		if modelCounts != 9 {
```

Comment:
It may not be enough to test the number of lists, but perhaps the entire structure can be tested.

Personally, in my opinion, this method `SetDefaultClusterResourceModels` is not so universal.

Reply:
ok